main.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-unused-vars */
+const homeLink = document.querySelector('#homeLink');
+const aboutLink = document.querySelector('#aboutLink');
+const speakersCards = document.querySelector('.speakers-elt');
+
```

Comment:
[REQUIRED]As the previous code reviewer suggested Please do no disable linters for they help use write clean code. Please kindly enable your eslint in your main.js file